Vagrantfile

```diff
@@ -8,16 +8,7 @@ Vagrant.configure(2) do |config|
 
   config.vm.synced_folder "~/.aws", "/home/vagrant/.aws"
 
-  # Need to use rsync in order to prevent a vboxfs/docker/gunicorn-related
-  # file corruption issue.
-  config.vm.synced_folder ".", "/vagrant",
-      type: "rsync",
-      rsync__exclude: [".git/",
-                       "reports/",
-                       "geoserver/data_dir/",
-                       "django/publicmapping/publicmapping/config_settings.py",
-                       "django/publicmapping/locale/"],
```

Comment:
I still haven't been able to replicate exactly the symptoms you and @pcaisse have been seeing, but I did run into similar issues after doing a `vagrant destroy` and then trying to run `./scripts/load_pa_data`. I was able to resolve those problems by changing this line to `"django/publicmapping/locale/*/*.mo"` . That way, the `.po` files are synced, but the compiled `.mo` files are not. I think this should also allow us to get pull out Rosetta-generated changes to the `.po` files by simply running `vagrant rsync-back`, rather than the manual `scp` approach. https://github.com/PublicMapping/DistrictBuilder#translations